catalog/couchdb/couchdb-cluster.bom

```diff
@@ -35,12 +35,22 @@ brooklyn.catalog:
       services:
       - type: org.apache.brooklyn.entity.nosql.couchdb.CouchDBNode
         brooklyn.config:
+          provisioning.properties:
+            customizer:
+              $brooklyn:object:
+                type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+                object.fields:
+                  enabled: $brooklyn:config("couchdb.sharedsecuritygroup.create")
+                  tcpPortRanges:
+                    - "1024-65535"
```

Comment:
Regretfully not, erlang picks some random ports to use. We could reduce it to a specified range as [detailed here](http://docs.couchdb.org/en/2.0.0/cluster/setup.html#firewall) but I went for this in the short term.